conans/model/info.py

```diff
@@ -193,6 +194,7 @@ def loads(text):
         result.full_options = OptionsValues.loads(parser.full_options)
         result.full_requires = RequirementsList.loads(parser.full_requires)
         result.requires = RequirementsInfo(result.full_requires, None)
+        result.recipe_hash = parser.recipe_hash
```

Comment:
ConfigParser returns empty string "" for non existing fields, maybe None is preferred here to indicate absence